src/components/CheckingCards.test.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import '@testing-library/jest-dom/extend-expect'
+import { render } from "@testing-library/react"
+
+import CheckingCards from "./CheckingCards";
+
+describe('CheckingCardsAccount test', () => {
+    it('Render checking cards', () => {
+        const component = render(<CheckingCards/>);
```

Comment:
CheckingCards should be able to receive an array of accounts. That way you could create an array here, pass it through props and do something similar to what you did in the other test.